modules/nw-ingress-controller-nodeportservice-projects.adoc

```diff
@@ -0,0 +1,136 @@
+    nodePlacement:
+            nodeSelector:
+              matchLabels:
+                ingresscontroller: "custom-ic3" <3>
+    namespaceSelector:
+     matchLabels:
+       ic: custom-ic3 <4>
+    endpointPublishingStrategy:
+      type: NodePortService 
+kind: List
+metadata:
+  resourceVersion: ""
+  selfLink: ""
+# ...
+----
+<1> Specify the `name` with a name for the `IngressController` object.
+<2> Specify the `domain` for the wilcard DNS record.
+<3> Specify the label for the `ingresscontroller` CR.
+<4> Matches routes from any namespaces with this label.
+
+. Add a label to a node by using the `oc label node` command:
++
+[source,terminal]
+----
+$ oc label node <node_name> ingresscontroller=<ingress_controller_custom_resource>
+----
+
+. Create the `Pod` object:
++
+[source,terminal]
+----
+$ oc create -f <pod_object_name>.yaml
+----
+
+. Locate the hostname by running the `oc get svc` command against the `openshift-ingress` namespace. The hostname of the load balancer is the `EXTERNAL-IP` associated with the `router-default` service in the `openshift-ingress` namespace.
+
+[source,bash]
+----
+$ oc get svc -n openshift-ingress
+NAME                        TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                                     AGE
+router-internal-custom-ic   ClusterIP   172.30.177.55    <none>        80/TCP,443/TCP,1936/TCP                     155m
+router-internal-default     ClusterIP   172.30.195.74    <none>        80/TCP,443/TCP,1936/TCP                     223d
+router-nodeport-custom-ic   NodePort    172.30.109.219   <none>        80:32432/TCP,443:31366/TCP,1936:30499/TCP   155m
+----
+
+. Use the `oc new-project` command to create a new project:
++
+[source,terminal]
+----
+$ oc new-project <ingress_controller_name>
+----
+
+. Enter the following command to label the additional Ingress Controller:
++
+[source,terminal]
+----
+$ oc label namespace <ingress_controller_name> ic=<ingress_controller_name>
+----
+
+. WHAT does this do?
++
+[source,terminal]
+----
+$ oc new-app --image=quay.io/openshifttest/hello-openshift:multiarch
+----
+
+. Set the domain for the Ingress Controller hostname:
++
+[source,terminal]
+----
+$ oc expose svc/hello-openshift --hostname=hello-openshift.<domain_name>
+----
+
+. Update the `IngressController` CR to exclude the namespace for the Ingress Controller from the default Ingress Controller:
++
+[source,terminal]
+----
+$ oc patch --type=merge -n openshift-ingress-operator ingresscontroller/default --patch '{"spec":{"namespaceSelector":{"matchExpressions":[{"key":"ic","operator":"NotIn","values":["<ingress_controller_name>","custom-ic2","custom-ic3"]}]}}}'
+----
+
+.Verification
+
+. Verify that the wildcard DNS record is managed by the Ingress Controller. A successful connection means that your cluster can use IP addresses from external DNS servers for DNS resolution.
++
+[source,terminal]
+----
+$ curl hello-openshift.<ingress_controller_name>.<domain_name>
```

Comment:
```suggestion
$ curl hello-openshift.custom-ic3.<domain_name>:32432
```

Reply:
We'll need to discuss using an actual example versus <user_replaced> values where the latter would be better from a docs perspective to help maintainability and prevent any customer feedback that might happen from time to time.